.gitignore

```
@@ -360,4 +359,4 @@ MigrationBackup/
 .ionide/
 
 # Fody - auto-generated XML schema
-FodyWeavers.xsd
```

It looks like only a newline was added here. If this change is unintentional, consider reverting it to avoid unnecessary diffs.

**Suggested change**
```suggestion
-FodyWeavers.xsd
\ No newline at end of file
```